app/models/observation/naming_consensus.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class Observation
+  class NamingConsensus
+    attr_accessor :obs
+    attr_accessor :namings
+    attr_accessor :votes
+
+    def initialize(obs)
+      @obs = obs
+      @namings = obs.namings
+      @votes = @namings.map(&:votes).flatten
+    end
+
+    ############################################################################
+    #
+    #  :section: Preferred Naming
+    #
+    ############################################################################
+
+    # Observation.user's unique preferred positive Name for this observation
+    # Returns falsy if there's no unique preferred positive id
+    # Used in show obs subtitle: owner_naming_line
+    def owner_preference
+      owner_uniq_favorite_name if owner_preference?
+    end
+
+    private
+
+    # Does observation.user have a single preferred id for this observation?
+    def owner_preference?
+      owner_uniq_favorite_vote&.value&.>= Vote.owner_id_min_confidence
+    end
+
+    def owner_uniq_favorite_name
+      favs = owner_favorite_votes
+      favs[0].naming.name if favs.count == 1
```

Comment:
```suggestion
      favs[0].naming.name if favs.one?
```
Just a style thing, so take it or leave it. Interesting that there's no cop for this.